src/rpc/rawtransaction.cpp

```diff
@@ -328,6 +337,10 @@ static UniValue gettxchainlocks(const JSONRPCRequest& request)
         }
         if (height != -1) {
             chainLock = llmq_ctx.clhandler->HasChainLock(height, hash_block);
+            result.pushKV("mempool", false);
+        }
+        else {
+            result.pushKV("mempool", true);
         }
```

Comment:
should preserve the same order for all cases and push `mempool` after `chainlock`